src/com/jambit/onboarding2020/tbrpg/domain/Room/MobRoom.java

```diff
@@ -3,86 +3,185 @@
+                enemy.attack(player);
+                evaluateFight(in, enemy, player);
             }
-            System.out.println("Treffer!");
-         } else {
-            System.out.println("Der Gegner hat den Angriff abgewehrt");
-         }
-      }
-
-      if (random.nextBoolean()) {
-         if (input == 2) {
-            System.out.println("Du hast den Angriff des Gegner abgewehrt!");
-         } else {
-            enemy.attack(Player.getPlayerInstance());
-            System.out.println("Du wurdest getroffen!");
-         }
-      }
-   }
-
-   private void evaluateFight(GameInput in, Enemy enemy) {
-      System.out.println("Lebenspunkte des Players: " + Player.getPlayerInstance().getHealthState());
-      System.out.println("Lebenspunkte des Enemy: " + enemy.getHealthState());
-   }
-
-   @Override
-   public void skip() {
-      System.out.println("Feigling...du überspringst den Raum.");
-   }
+        }
+    }
+
+
+    private void evaluateFight(GameInput in, Enemy enemy, Player player) {
+        System.out.println("|Lebenspunkte des Spielers: " + Player.getPlayerInstance().getHealthState() + "|");
+        System.out.println("|Lebenspunkte des Gegners: " + enemy.getHealthState() + "|");
+        chooseItem(player);
+        System.out.println("               _                          _ \n" +
+                "  _ _  _____ _| |_   _ _ ___ _  _ _ _  __| |\n" +
+                " | ' \\/ -_) \\ /  _| | '_/ _ \\ || | ' \\/ _` |\n" +
+                " |_||_\\___/_\\_\\\\__| |_| \\___/\\_,_|_||_\\__,_|");
+    }
+
+    private void chooseItem(Player player) {
+        System.out.println("Möchstest du ein Item einsetzen? " +
+                "\n Wenn ja, tippe Heiltrank [ht] oder Fluchtseil [fs]" +
+                "\n ansonsten, tippe Nein [n]");
+
+        String input = scan.nextLine();
+        while (!(input.equalsIgnoreCase("Heiltrank")) && !(input.equalsIgnoreCase("ht"))
+                && !(input.equalsIgnoreCase("Fluchtseil")) && !(input.equalsIgnoreCase("fs"))
+                && !(input.equalsIgnoreCase("nein")) && !(input.equalsIgnoreCase("n"))) {
+            System.out.println("Falsche Eingabe, bitte erneut eintippen");
+            input = scan.nextLine();
+        }
+
+        if (input.equalsIgnoreCase("Heiltrank") || input.equalsIgnoreCase("ht")) {
+            if (Inventory.size() == 0)
+                System.out.println("Dein Inventar ist leer!");
+            else if (checkHealthPotion()) {
+                System.out.println("Du setzt einen Heiltrank ein");
+                player.getConsumableFromInventory("Heiltrank").consume();
+            } else {
+                System.out.println("Du hast keinen Heiltrank!");
+            }
+
+
+            if (input.equalsIgnoreCase("Fluchtseil") || input.equalsIgnoreCase("fs"))
+                if (Inventory.size() == 0)
+                    System.out.println("Dein Inventar ist leer!");
+                else if (checkEscRope()) {
```

Comment:
Der Fall wird in der Methode GameEngine.interactWithInventory() behandelt, dort findest du eine etwas verkürzte Implementierung von dieser Abfrage:
```
if (player.getConsumableFromInventory("Heiltrank") != null) {
               player.getConsumableFromInventory("Heiltrank").consume();
            } else {
               System.out.println("Du hast keinen Heiltrank im Inventar!");
            }
```